src/mqttcm_connect.c

```diff
@@ -427,28 +426,35 @@ void on_connect(struct mosquitto *mosq, void *obj, int reason_code, int flag, co
 	MqttCMInfo("on_connect: success. broker_connect set to 1\n");
 	broker_connect = 1;
 
+	//For Mqtt reconnection state use in-cache subscribe list
```

Comment:
//For Mqtt reconnection case, use in-memory subscribe list to subscribe the components again